corehq/apps/userreports/sql.py

```diff
@@ -114,7 +123,17 @@ def _hash(parts):
         end = parts[-1]
         return hashlib.sha1('{}_{}'.format(hashlib.sha1(front).hexdigest(), end)).hexdigest()[:8]
 
-    return "_".join(parts + [_hash(parts)])
+    column_len = 0
+    new_parts = []
+    parts.reverse()
+    for part in parts:
```

Comment:
Could this loop be dropped by doing something like `path = path[-54:]` before the `parts = path.split("/")` above?